### ISSUE TITLE
Quotation marks added around strings for no apparent reason

### ISSUE THREAD

**Issue by gmestdacOLV** [mage-ai/mage-ai#2829]
When loading data from excel, then transforming it using an sql server connection the contents of string columns automatically get surrounded by quotation marks. I see no immediate reason for this change to the data.
I tried to capture this in a screenshot, where you can see that for example "Diagnoses Uro" is loaded as-is from the file, without quotation marks, but is then written to the SQL database with quotation marks added.
This is not a matter of it simply being displayed with quotation marks around it, as the actual values in the output table also have the quotation marks. 
Thanks for looking into this. 

![image](https://github.com/mage-ai/mage-ai/assets/123754865/f222e114-4602-4cee-832c-4ac160c84417)

**Comment by wangxiaoyou1993** [comment]
Will look into this today.

**Comment by omkar-foss** [comment]
Hey @gmestdacOLV @wangxiaoyou1993, I was able to reproduce this issue successfully in my local Mage setup with source as the [Titanic CSV](https://github.com/datasciencedojo/datasets/blob/master/titanic.csv) and destination as a table in SQL Server 2022.

The extra surrounding double quotes are getting added while performing string conversion of the data [here in code](https://github.com/mage-ai/mage-ai/blob/master/mage_ai/io/mssql.py#L117-L124), specifically by the `json.dumps()` which adds double quotes to the value if the value of the underlying `object` is already a string.

I've raised a [PR](https://github.com/mage-ai/mage-ai/pull/2944) to strip the extraneous pair of double quotes, intentionally keeping it outside the `if-elif` scope so that it remains as a general check for removal of double quotes from data before saving to SQL Server, and also ensures that any future changes to the `if-elif` block do not reintroduce this issue.

**Note:** I tried to reproduce this issue with destination as MySQL as well as Postgres, but this issue isn't occurring in those. Also checked both code implementations and looks like they've different kind of handling for the data before insertion.
